src/main/java/org/broadinstitute/consent/http/models/DarCollectionSummary.java

```diff
@@ -16,6 +16,9 @@
 
 public class DarCollectionSummary {
 
+  public static final String FILTER_DRAFT_DAR_COLLECTIONS =
+          "EXISTS (SELECT 1 FROM data_access_request WHERE (collection_id = c.collection_id and draft = false))";
```

Comment:
Nitpick: I kind of find this language confusing - `filtering` i expect to return all results that match the filter, but `find` is where i expect a single result. The variable name (with the "s" on collections) also implies multiple results possible.

(Sorry, I've mostly had experience working on nosql databases, so if I'm wrong about this, please let me know!) Is this supposed to return a single result?

Reply:
Yeah totally! I agree with this feedback and above that it should just be inline with the DAO query string, so that makes the confusing name issue moot.